src/main/java/com/shopping_app/shoppingApp/service/UserService.java

```diff
@@ -0,0 +1,17 @@
+package com.shopping_app.shoppingApp.service;
+
+import com.shopping_app.shoppingApp.model.Request.UserLoginRequest;
+import com.shopping_app.shoppingApp.model.Request.UserProfileUpdateRequest;
+import com.shopping_app.shoppingApp.model.Request.UserRegisterRequest;
+import com.shopping_app.shoppingApp.model.Response.UserLoginResponse;
+import com.shopping_app.shoppingApp.model.Response.UserResponse;
+import java.util.List;
+
+public interface UserService {
```

Comment:
Why do we need an interface for the service class?

Reply:
we can easily found what are all the  methods and logic added for service  class, rather checking in implementation class , also we can reuse the interface if needed

Reply:
removed interface in service class